spec/registry.md

```diff
@@ -826,3 +812,24 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 > The form of these serializations is known and is a de facto standard.
 > Support for these extensions is expected to be required in the post-tech preview.
 > See: https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/
+
+### Date and Time Locale Options
+
+Implementations MAY support overriding some of the options set by the current locale
+or an implementation-defined date/time _operand_ value.
+To do so, the following **_<dfn>date/time locale options</dfn>_**
+can be made available on `:datetime`, `:date`, and `:time`:
+
+- `calendar`
+  - valid [Unicode Calendar Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCalendarIdentifier)
+- `numberingSystem`
+   - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
+- `hour12`
+  - `true`
+  - `false`
+- `timeZone`
+  - valid identifier per [BCP175](https://www.rfc-editor.org/rfc/rfc6557)
```

Comment:
Don't link the RFC. Link the BCP (e.g. https://www.rfc-editor.org/bcp/bcp175)... but... time zone identifiers have a ton of quirks in them. Also, we almost certainly want to allow offset time zones (e.g. `GMT-01:23`) and we may want to allow special sauce like metazones. CLDR has a bunch of stuff about this, but I'm too busy this morning to look up the precise reference. It's somewhere near [here](https://www.unicode.org/reports/tr35/tr35-dates.html#Time_Zone_Names)

Reply:
Happy to go with whatever; this reference was not changed from the earlier one we already included.

Reply:
Just for the record, a narrow-ish well-formed EBNF of current and past timezone IDs would be:

```
tzId   := tzPath | tzEtc | tzOld
tzPath := tzPart ("/" tzPart)+;
tzPart := tzWord ("_" tzWord)*;
tzWord := [A-Z][a-z]*;
tzEtc  := "Etc/" ("UTC" | "GMT" ([+\-] \d{1,2})?)
tzOld  := HST" | "PST8PDT" | "MST" | "MST7MDT" | "CST6CDT" | "EST" | "EST5EDT" | "WET" | "CET" | "MET" | "EET" | "Factory"
```

A very loose well-formed EBNF would be

```
tzId  := [a-zA-Z0-9+\-/_]+
```

Reply:
My tendency is to want a version of the narrow-ish EBNF (in ABNF, since that's the dialect our WG uses). It wouldn't want to support `tzOld` and would want to add support for "UTC" as an explicit zone name. See suggestion thread below.